File: solutions/215/solution.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        minheap = []
        for num in nums:
            if len(minheap) < k:
                heapq.heappush(minheap,num)
            elif num > minheap[0]:
                heapq.heappop(minheap)
                heapq.heappush(minheap,num)
        return minheap[0]

File: solutions/215/solution.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int> heap;
        for(int num : nums){
            if(heap.size()<k){
                heap.push(-1*num);
            } else {
                if(num > -1*heap.top()){
                    heap.pop();
                    heap.push(-1*num);
                }
            }
        }
        return -1*heap.top();
    }
};